packages/elements-core/src/utils/http-spec/IServer.ts

```diff
@@ -26,15 +32,21 @@ export const getServersToDisplay = (originalServers: IServer[], mockUrl?: string
   return servers;
 };
 
-export const getServerUrlWithDefaultValues = (server: IServer): string => {
+export const getServerUrlWithDefaultValues = (server: IServer): string | null => {
```

Comment:
Making it nullable will prob have downstream consequences right? Not end of world, but we should be aware as we update in elements and elsewhere - hopefully the typings catch it.